### DISCUSSION TITLE
define styles for tag

### DISCUSSION THREAD

**Discussion by Kimzify** [chakra-ui/panda#2014] [General]
### Description

according to [panda's documentation](https://panda-css.com/docs/concepts/writing-styles#native-css-nesting) it is possible to define styles for nested tag like this:
`    '& span': {
      color: 'pink.400'
    }
`

I have a component that accept children as a svg component, and I want to style the svg, but it doesn't work at all. 
```
      <span
        className={cx(stepClassNames.bullet, classNames?.bullet)}
        data-active={state === 'active'}
        data-completed={state === 'completed'}
        data-disabled={state === 'disabled'}
      >
        {type === 'complex' && children}
      </span>

```

```
    bullet:{
      '& svg': {
        fontSize: 'initial',  // not working 
       },
     }


```

I think it's because children component is not specified and it cannot generate that style. 
is there any solution or recommendation here? 

### Link to Reproduction

--

### Panda CSS Version

0.22.0

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
Hi @Kimzify,

I can't reproduce this
<img width="1719" alt="CleanShot 2024-01-16 at 18 57 13@2x" src="https://github.com/chakra-ui/panda/assets/6916170/cbe44881-6178-4cba-a1f5-098b419e82ed">

Please create a reproduction before we can consider this an issue

I think your code is correct, but a reproduction will help verify that.
